bookkeeper-server/src/main/java/org/apache/bookkeeper/client/SyncCounter.java

```diff
@@ -22,6 +22,7 @@
 package org.apache.bookkeeper.client;
 
 import java.util.Enumeration;
+import org.apache.bookkeeper.client.api.LedgerEntry;
```

Comment:
this seems not necessary

Reply:
Sync Counter is not used by current code, it is here only for backward compatibility.
Did you run all rests, even compatibility?

Reply:
Sync Counter is not used by current code, it is here only for backward compatibility.
Did you run all rests, even compatibility?

Reply:
Thanks. will re-run the compatibility tests.

Reply:
re-checked, the BC tests all passed.